app/views/contexts/summary/list_collector_block.py

```diff
@@ -160,20 +163,27 @@ def _add_link(
             )
 
     def _get_related_answers(
-        self, list_model: ListModel
-    ) -> Optional[dict[str, list[dict[str, Any]]]]:
+        self, list_model: ListModel, repeating_blocks: Sequence[ImmutableDict]
+    ) -> dict[str, list[dict]] | None:
         section_id = self._section["id"]
 
         related_answers = self._schema.get_related_answers_for_list_for_section(
             section_id=section_id, list_name=list_model.name
         )
-        if not related_answers:
+
+        blocks: list[dict] = []
+
+        if related_answers:
+            blocks += self._get_blocks_for_related_answers(related_answers)
+
+        if list_model:
```

Comment:
Does list_model not always exist if you always pass a list_model with type ListModel into this method?

Reply:
Default behaviour when `__len__` is implemented, but `__bool__` is not, is to evaluate as `len(list_model) != 0`, which what we're checking, if there are any list items. 
https://blog.finxter.com/python-__len__-magic-method/ - can't find the PEP standard for it though. 

I'll change to `len(list_model) != 0` to be more clear

Reply:
Done 👍